api-report/container-runtime-definitions.api.md

```diff
@@ -70,7 +70,7 @@ export interface IContainerRuntimeEvents extends IContainerRuntimeBaseEvents {
     // (undocumented)
     (event: "codeDetailsProposed", listener: (codeDetails: IFluidCodeDetails, proposal: IPendingProposal) => void): any;
     // (undocumented)
-    (event: "dirtyDocument" | "dirty" | "disconnected" | "dispose" | "savedDocument" | "saved", listener: () => void): any;
+    (event: "dirtyDocument" | "dirty" | "disconnected" | "dispose" | "savedDocument" | "saved" | "attached", listener: () => void): any;
```

Comment:
Is this a safe change with respect to back-compat? Are there any scenarios where developers _must_ handle the "attached" event?

Reply:
No. This is only used within ContainerRuntime, so back compat is not an issue.